package.json

```diff
@@ -4,8 +4,9 @@
   "description": "Source for building https://dist.ipfs.io.",
   "main": "",
   "scripts": {
-    "lint": "standard scripts/*.js site/assets/js/*.js && shellcheck *.sh",
-    "clean": "rm -f site/static/*.js site/static/*.css",
+    "standard": "standard scripts/*.js site/assets/js/*.js",
```

Comment:
extracted to it's own script so we can run `npm run standard -- --fix` and have it auto fix the lints.